src/components/TimeEstimator/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Time estimator.
+ * @copyright IBM Security 2019
+ */
+
+import Time16 from '@carbon/icons-react/lib/time/16';
+
+import classnames from 'classnames';
+import { node, string } from 'prop-types';
+import React from 'react';
+
+import { getComponentNamespace } from '../../globals/namespace';
+
+const namespace = getComponentNamespace('time-estimator');
+
+/**
+ * Time estimators display the estimated amount of time it takes to complete a flow.
+ */
+const TimeEstimator = ({ children, className, ...props }) => (
```

Comment:
I wonder about calling this `TimeEstimator` because it kinda insinuates that the component is doing more than positioning an icon next to a string? 🤔 Can we just call it something else?

OR it would even be make more generic, really. Since it's not doing anything logic-wise. It could just be text next to any ol' icon. 🤔

Reply:
Good points! 

I think the 'text beside a time icon indicating how long a flow is going to take' is a common pattern so I have concerns with making it too generic, but I agree that the name could be misleading - perhaps `TimeIndicator` is more appropriate.

Would you have any suggestions? I'm also tagging @petervachon for input.

Reply:
Oh yeah I like `TimeIndicator` more! 👀 
@petervachon what do you think?